app/parse.py

```diff
@@ -15,8 +20,95 @@ class Product:
     num_of_reviews: int
 
 
-def get_all_products() -> None:
-    pass
+def setup_driver():
+    options = Options()
+    options.add_argument("--headless")
```

Comment:
Using `--headless` mode is efficient for running scripts without a GUI, but ensure that the environment supports it. Consider adding a check or handling exceptions that might occur if the headless mode is not supported.